analyses/snv-callers/.gitignore

```diff
@@ -1,3 +1,4 @@
 # ignore folders with big results files
 results
-ref_files
+ref_files/*
+!ref_files/gencode.v19.basic.exome.hg38liftover.bed
```

Comment:
For now, this needs to be added in somewhere, but I think it will be in a future data release.